ui/library/src/utils/MaxVisibleElement.ts

```diff
@@ -0,0 +1,17 @@
+import { Nullable } from '../components/types/utils';
+
+type VisibleElementKey = Nullable<number | string>;
```

Comment:
I think the name of this type should be consistent with what below function does, maybe changing to `type VisibleElement` would be better. Or maybe defining this type under `types` and import the type from there would be even better.